CHANGELOG.md

```diff
@@ -0,0 +1,207 @@
+# Changelog
+
+Darts is still in an early development phase and we cannot always guarantee backwards compatibility. Changes suceptible to **break existing code** working on a previous release are marked with a "&#x1F534;". Sorry for the inconvenience!
+
+## [Unreleased](https://github.com/unit8co/darts/tree/develop)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.3.0...develop)
+
+## [0.3.0](https://github.com/unit8co/darts/tree/0.3.0) (2020-10-dd)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.2.3...0.3.0)
+
+### For users of the library:
+**Added:**
+
+- Indexing on TimeSeries [\#150](https://github.com/unit8co/darts/pull/150)
+- New `FourTheta` forecasting model [\#123](https://github.com/unit8co/darts/pull/123), [\#156](https://github.com/unit8co/darts/pull/156)
+- `map()` method for TimeSeries [\#121](https://github.com/unit8co/darts/issues/121), [\#166](https://github.com/unit8co/darts/pull/166)
+- Further improved Multivariate TimeSeries capabilities:
+  - Added multivariate support to the Neural Network based forecasting models, namely `TorchForecastingModel` and its subclasses (`TCNModel` and `RNNModel`)
```

Comment:
Strictly speaking multivariate support is already present in 0.2.2, just with a different syntax.